addons/base_import_module/tests/test_import_module.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import base64
+
+import odoo.tests
+from odoo.tools import file_open
+
+@odoo.tests.tagged('post_install', '-at_install')
+class TestImportModule(odoo.tests.TransactionCase):
+
+    def test_import_and_uninstall_module(self):
+        IrModule = self.env['ir.module.module']
+        module = 'test_module'
+        bundle = 'web.assets_backend'
+        path = '/test_module/static/src/js/test.js'
+        file_content = b"console.log('test_module javascript');\n"
+
+        # Import test module
+        with file_open('base_import_module/tests/test_module.zip', 'rb') as zip_file:
```

Comment:
I'd rather not have completely unnecessary opaque binary files committed and loaded when that's not necessary. The zip can easily be created on the fly in-memory from visible test data (which can be committed as sibling files if having them as literal string in the test file is a bit too verbose).